android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```diff
@@ -201,7 +216,9 @@ void getUserMedia(
 
             CameraCaptureController cameraCaptureController = new CameraCaptureController(
                 cameraEnumerator,
-                videoConstraintsMap);
+                videoConstraintsMap,
+                cameraUvcStrategy
```

Comment:
can't we pass the enumerator instead? feels more natural to have 1 "meta" enumerator.

Reply:
Sorry, that I didn't understand.

Reply:
I think we can create the strategy inside the enumerator, and provide a getter for it.